templates/about.html

```diff
@@ -1,113 +1,165 @@
 <div class="container">
   <div class="row">
-  <div class="col-md-6">
-    <p style="margin-top: 6px;">
+    <div class="col-md-6">
+      <p style="margin-top: 6px;">
 
-      <span>Everyone loves the&nbsp;</span><a target="_blank" href="https://developers.google.com/speed/libraries/?csw=1">Google CDN</a><span>&nbsp;right? Even&nbsp;</span><a href="https://www.asp.net/ajax/cdn" target="_blank">Microsoft runs a public CDN</a>.
-      <br><span>The problem is, they only host the most popular libraries.</span>
-      <br><span>We host all the popular libraries - JavaScript, CSS, SWF, images, etc!</span>
-      <br><span>Join us now on GitHub!</span><br>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=watch&count=true&v=2" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=star&count=true" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=new-website&type=star&count=true" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      <iframe src="https://ghbtns.com/github-btn.html?user=cdnjs&repo=cdnjs&type=fork&count=true" frameborder="0" scrolling="0" width="105px" height="20px"></iframe>
-      </p><p>
-      <span>Supports <a href="https://http2.github.io" target="_blank"><strong>http2.0</strong>/</a>http/https/<a href="https://en.wikipedia.org/wiki/SPDY" target="_blank">spdy</a>, we encourage always using <strong>https</strong>!
-      <br /><span>Currently, CDNJS is one of the most famous free and public web front-end CDN services which is used by ~2,999,000 websites worldwide, see more detail on <a href="https://trends.builtwith.com/cdn/CDN-JS" target="_blank">BuiltWith</a> and <a target="_blank" href="https://w3techs.com/technologies/overview/content_delivery/all">W3Techs</a>.</span>
-    </p>
-
-    <div class="logged-in" style="display: none;">
+        Everyone loves the&nbsp;<a target="_blank" href="https://developers.google.com/speed/libraries/?csw=1">Google
+        CDN</a>&nbsp;right? Even&nbsp;<a href="https://www.asp.net/ajax/cdn" target="_blank">Microsoft runs a public CDN</a>.
+        <br/>The problem is, they only host the most popular libraries.
```

Comment:
same here

Reply:
What is the issue here? @PeterDaveHello

Reply:
Oh, the space ;)

```diff
- <br/>
+ <br />
```

Reply:
@PeterDaveHello Why are we adding spaces to all self-closing tags? That seems very non-standard to me and a waste of data?

Reply:
It's about readability. Just like the other spaces in indentation.

Reply:
I don't see how it improve readability, having a space before the `/>` seems irrelevant? It just means every time someone auto-formats this code, they have to manually put all the spaces back in?

Reply:
Your code formatter should be configurable to keep the space, otherwise, replace it with another one, please.

Reply:
@MattIPv4 do you have any idea why we use `<br/>` but not `<br>` here?

Reply:
@PeterDaveHello We use `<br/>` for xhtml compliancy. See https://www.w3.org/TR/xhtml1/#h-4.6

Reply:
so ... https://www.w3.org/TR/xhtml1/#C_2

Reply:
@PeterDaveHello Yup, yet their above examples in 4.6 don't have that space. I'll update webstorm to use spaces.

Reply:
It's more a styling issue now, we should try to maintain the more readability styling, not only here, also the other places, like commit and commit message, and in the other languages, that's why I ask if you can add some checks in the brand repo for the python codes, this is a very important issue for collaborate.